.gitpod.yml

```diff
@@ -0,0 +1,29 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:  
+  - name: wasmCloud
+    init: cargo install --path . 
+    command: wasmcloud --version
+  - name: wash
+    init: cargo install wash-cli --force
+    command: wash --version
```

Comment:
We may want to wait to merge the gitpod config until `0.4.0` is released for `wash-cli`. It also brings up a valid point that the latest released version of `wash` should ALWAYS match the latest release of `wasmcloud-host` + `wasmcloud-control-interface`

Reply:
`wash-cli` `0.4.0` will be released once my no-nats PR is merged

Reply:
We can either merge this now and just update the files once wash is released or we can wait.. I can be convinced either way.

Reply:
I think we should install from the latest release regardless, but I think we should wait so nobody hops in and tries to develop in an environment that isn't fully working yet. Apologies on letting `wash` get so far out of sync